crates/sui-indexer/migrations/2022-11-18-195259_transactions/up.sql

```diff
@@ -1,12 +1,16 @@
-CREATE TABLE transactions
-(
+-- SuiAddress and ObjectId type, 0x + 64 chars hex string
+CREATE DOMAIN address VARCHAR(66);
+-- Max char length for base58 encoded digest
+CREATE DOMAIN base58digest VARCHAR(44);
```

Comment:
move the type definition here b/c migrations are run sequentially, otherwise events cannot use this type.